app/models/dropdown_select_facet.rb

```diff
@@ -0,0 +1,48 @@
+class DropdownSelectFacet < FilterableFacet
+  attr_writer :value
+
+  def allowed_values
+    facet['allowed_values']
+  end
+
+  def options
+    allowed_values.map do |allowed_value|
+      {
+        value: allowed_value['value'],
+        text: allowed_value['text'],
+        selected: selected_value == allowed_value,
+      }
+    end
+  end
+
+  def name
+    "Filter by #{facet['name']}"
```

Comment:
This is a view concern. Can we extract to a presenter? How might we handle localisation?